web/src/app/user/user.component.html

```diff
@@ -1,25 +1,87 @@
 <div class="container">
   <div class="row">
     <div class="col-md-4 col-md-offset-4">
-      <div class="card" style="width: 20rem;">
+      <div class="card" style="width: 30rem">
         <img class="card-img-top image" [src]="user.image" alt="Card image cap">
         <div *ngIf='user.provider != "password"'class="card-body">
           <h4 class="card-title">{{user.name}}</h4>
           <p class="card-text">This is an example of Social Authentication using Firebase in an Angular 5 web app. </p>
         </div>
         <div *ngIf='user.provider == "password"'class="card-body">
-          <form [formGroup]="profileForm">
+          <!--<form [formGroup]="profileForm">
```

Comment:
If the code is not going to be used, then it should probably be removed instead of just commented out.